jax/lax_linalg.py

```diff
@@ -117,6 +121,48 @@ def cholesky_cpu_translation_rule(c, operand):
 
 xla.backend_specific_translations['cpu'][cholesky_p] = cholesky_cpu_translation_rule
 
+# Asymmetric eigendecomposition
+
+def eig_impl(operand):
+  w, vl, vr = xla.apply_primitive(eig_p, operand)
+  return core.pack((w, vl, vr))
```

Comment:
I believe you can shorten this to just

```
return xla.apply_primitive(eig_p, operand)
```

That is, the application of the primitive will return a JaxTuple (or a DeviceTuple, which is to a JaxTuple what a DeviceArray is to a numpy.ndarray), so there's no need to unpack it and then re-pack it.